config/environments/production.rb

```diff
@@ -51,6 +51,10 @@
           "config.log_level = :debug",
           'config.log_level = ENV.fetch("LOG_LEVEL", "info").to_sym'
 
+gsub_file "config/environments/production.rb",
+          "ActiveSupport::Logger.new(STDOUT)",
+          "ActiveSupport::Logger.new($stdout)"
+
```

Comment:
Because of rubocop error
```
config/environments/production.rb:141:50: C: Style/GlobalStdStream: Use $stdout instead of STDOUT. (https://rubystyle.guide#global-stdout)
    logger           = ActiveSupport::Logger.new(STDOUT)
                                                 ^^^^^^
```

Reply:
🐘 would be awesome to break this out into its own PR as it'll block everyone's PRs (including mine #198) and this one still needs review.